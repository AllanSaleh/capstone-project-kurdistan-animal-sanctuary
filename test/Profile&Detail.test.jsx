import React from "react";
import renderer from "react-test-renderer";
import { test, expect } from "@jest/globals";
import ProfileandDetail from "../components/Profile&Detail";

test("proparly renders the Profile&Detail component", () => {
  const tree = renderer.create(<ProfileandDetail />).toJSON();
  expect(tree).toMatchSnapshot();
});
