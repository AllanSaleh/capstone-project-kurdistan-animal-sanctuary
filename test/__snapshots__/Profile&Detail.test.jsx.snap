// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proparly renders the Profile&Detail component 1`] = `
<div
  className="flex flex-wrap md:flex-nowrap  py-14 "
>
  <div
    className="block md:flex md:flex-row "
  >
    <div
      className=" w-40 md:w-72 h-40 md:h-72 ml-5 rounded-full"
    >
      <img
        alt="animal profile "
        className="  w-36 md:w-60 h-36 md:h-60 border-2 border-secondary p-2 rounded-full"
      />
    </div>
    <div>
      <div
        className="w-48 "
      >
        <div
          className="px-5 py-4 font-bold text-primary text-2xl"
        />
        <div
          className="flex flex row text-md md:text-xl text-gray-700 text-opacity-50 font-medium"
        >
          <div
            className="pl-5  "
          >
            <ul>
              <li
                className="pb-5 "
              >
                Sex
              </li>
              <li
                className="pb-5 "
              >
                Age
              </li>
              <li
                className="pb-5 "
              >
                Vaccinated
              </li>
              <li
                className="pb-5 "
              >
                Shelter
              </li>
            </ul>
          </div>
          <div
            className="pl-12 "
          >
            <ul
              className="w-48"
            >
              <li
                className="pb-5 "
              />
              <li
                className="pb-5 "
              >
                 Years old 
              </li>
              <li
                className="pb-5 "
              />
              <li
                className="pb-5 "
              />
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="grid grid-flow-col w-4/5  "
  >
    <div
      className=" justify-self-end pt-5 md:pt-12"
    >
      <button
        className="mr-3 w-48 h-12 text-primary bg-highlight hover:bg-darken rounded-lg font-medium"
        type="button"
      >
        Apply For Adoption
      </button>
    </div>
  </div>
</div>
`;
